Hi @traversaro 
Thanks for this PR.
I noticed the regressions on Windows platform like this:
```
Starting package 31/31: sdformat6:x64-windows
Building package sdformat6[core]:x64-windows...
Could not locate cached archive: C:\vsts\_work\4\s\archives\3d\3d924945b47fca0a9318848ebcf70b1cb1564b5f.zip
-- Using cached C:/vsts/_work/4/s/downloads/osrf-sdformat-sdformat6_6.2.0.tar.gz
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:99 (message):
  

  File does not have expected hash:

          File path: [ C:/vsts/_work/4/s/downloads/osrf-sdformat-sdformat6_6.2.0.tar.gz ]
      Expected hash: [ 3d139ec4b4c9fbfd547ed8bfca0adb5cdca92c1b7cc4d4b554a7c51ccf755b9079c26a006ebfedc5bc5b1ba5e16ad950bb38c47ea97bf97e59a2fd7d12d60620 ]
        Actual hash: [ 3e3934010438bffbf10c1df29bd486c098e3c1bdf2b0349b69a53fb6f4d2bd3b3c8c4b4a8dfb413da13a638c0794f41c1bff4adb11a889b1552d90ba8b94c495 ]

  Please delete the file and retry if this file should be downloaded again.
```
It seems that the hash value should be kept as the before one.
Could you please check it again?

--- a/ports/sdformat6/CONTROL
+++ b/ports/sdformat6/CONTROL
@@ -3,3 +3,4 @@ Version: 6.2.0-1
 Homepage: http://sdformat.org/
 Build-Depends: boost-any, boost-variant, ignition-math4, urdfdom, tinyxml
 Description: Simulation Description Format (SDF) parser and description files.
+Supports: !(arm|uwp)

--- a/ports/sdformat6/portfile.cmake
+++ b/ports/sdformat6/portfile.cmake
@@ -1,4 +1,4 @@
-add vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
+vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH

--- a/ports/sdformat6/portfile.cmake
+++ b/ports/sdformat6/portfile.cmake
@@ -1,4 +1,4 @@
-include(vcpkg_common_functions)
+add vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
@@ -49,7 +49,4 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include
                     ${CURRENT_PACKAGES_DIR}/debug/share)
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/sdformat6 RENAME copyright)
-
-# Post-build test for cmake libraries
-vcpkg_test_cmake(PACKAGE_NAME SDFormat)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

